docs/book/features/block_volume.md

```diff
@@ -0,0 +1,278 @@
+
+* Wait for CNS to create a `PersistentVolume`
+
+* Verify if the status of `PersistentVolumeClaim` set to `Bound`
+
+## Static Volume Provisioning
+
+If you have an existing persistent storage device in your VC, you can use static provisioning to make the storage
+instance available to your cluster.
+
+### How does it work
+
+Static provisioning is a feature that is native to Kubernetes and that allows cluster administrators to make existing
+storage devices available to a cluster.
+
+As a cluster administrator, you must know the details of the storage device, its supported configurations, and mount options.
+
+To make existing storage available to a cluster user, you must manually create the storage device, a `PeristentVolume`,
+and a `PersistentVolumeClaim`.
+Because the PV and the storage device already exists, there is no need to specify a storage class name in the PVC spec.
+One of the ways to create the `PV` and the `PVC` binding is by using Label matching.
+
+Static Volume Provisioning is supported only in Vanilla Kubernetes clusters and across guess clusters.
+However, in Project Pacific, it's not supported.
+
+**NOTE:**
+  User is required to wait for some time after creating the PV and have the metadata-syncer mark this static volume as a
+  container volume before creating a POD that uses this volume. Otherwise, he/she will hit this issue - [2535120](https://bugzilla.eng.vmware.com/show_bug.cgi?id=2535120)
+
+### When do I use static provisioning
+
+Following are the common use cases for static volume provisioning:
+
+* **Use an existing storage device:**
+    You provisioned a persistent storage(FCD) directly in your VC and want to use this `FCD` in your cluster.
+
+* **Make retained data available to the cluster:**
+    You provisioned a volume with a `reclaimPolicy: retain` in the storage class by using dynamic provisioning.
+    You removed the `PVC`, but the `PV`, the physical storage in the VC, and the data still exist.
+    You want to access the retained data from an app in your cluster.
+
+* **Share persistent storage across namespaces in the same cluster:**
+    You provisioned a `PV` in a namespace of your cluster. You want to use the same storage instance for an app pod
+    that is deployed to a different namespace in your cluster.
+
+* **Share persistent storage across clusters in the same zone:**
+    You provisioned a `PV` for your cluster. To share the same persistent storage instance with other clusters in
+    the same zone, you must manually create the `PV` and matching `PVC` in the other cluster.
+
+**NOTE:**
+Sharing persistent storage across clusters is available only if the cluster and the storage instance are located in
+the same zone.
+
+### Steps to Statically Provision a Block Volume on Vanilla Kubernetes Cluster
+
+This section describes the step by step instructions to provision a PersistentVolume statically on a `Vanilla Kubernetes`
+Cluster
+
+* Define a PVC and a PV as shown below
+
+```yaml
```

Comment:
tab this yaml section to align under bullet - Define a PVC and a PV as shown below